app/views/completeOrderV.js

```diff
@@ -22,7 +22,7 @@ module.exports.createPrompt = (total, opts) => {
       pattern: optionsRegEx,
```

Comment:
`description: `\nAvailable Payment Options:\n${optionsMsg}\nChoose a payment option`,`

Reply:
i think the description on line 21 here, should be the above statement